src/pages/PersonalProfile/ProfileCard.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from '../../components/UI/Button/Button';
+import styled from 'styled-components';
+import './ProfileCard.css';
+
+const Card = styled.div`
+    width: 20%;
```

Comment:
What these 2 rules do?
`width: 20%;
 height: 90%;`

Reply:
width: 20% is messing up with responsiveness.
![width-rule-problem](https://user-images.githubusercontent.com/38657595/61593797-d541c680-abec-11e9-93e7-f596a4f35cf3.jpg)